.eleventy.js

```diff
@@ -0,0 +1,37 @@
+// This escapes the HTML inside the example code block
+function escapeHtml(unsafe) {
+  return unsafe
+       .replace(/\<br\>/g, "\n")
+       .replace(/&/g, "&amp;")
+       .replace(/</g, "&lt;")
+       .replace(/>/g, "&gt;")
+       .replace(/"/g, "&quot;")
+       .replace(/'/g, "&#039;");
+}
```

Comment:
Is this home-rolled, or copied from somewhere (and if so, where)? My immediate instinct is that this is an XSS waiting to happen.

Reply:
This is home-rolled, but I don't really see, how this should open up for an XSS, as it is only run once on the server side when building the documentation.